core/types/log.go

```diff
@@ -55,12 +54,12 @@ type Log struct {
 	Removed bool `json:"removed"`
 }
 
-type logMarshaling struct {
-	Data        hexutil.Bytes
-	BlockNumber hexutil.Uint64
-	TxIndex     hexutil.Uint
-	Index       hexutil.Uint
-}
+// type logMarshaling struct {
```

Comment:
why？

Reply:
because it is not used, checks by code style.